ruby/lib/trailer_vote/media_types/configuration.rb

```diff
@@ -11,9 +11,36 @@ module MediaTypes
     # Media Types for Configuration
     #
     class Configuration < BaseText
-      media_type 'configuration', defaults: { suffix: :json, version: 3 }
+      media_type 'configuration', defaults: { suffix: :json, version: 4 }
 
       validations do
+
+        version 4 do
+          attribute :configuration do
+            link :self
+            link :place
+            link :push_manifest
+            link :products
+            link :product_lookup
+            link :persona do
+              attribute :href, Types.makeFormattedUrl(:uuid)
+              attribute :templated, TrueClass
+            end
+            link :analytics do
+              attribute :href, Types::InfluxDbConnectionUrl
+            end
+            link :telemetrics do
+              attribute :href, Types::InfluxDbConnectionUrl
+            end
+            link :issues
+            link :persona_config do
+              attribute :href, Types.makeFormattedUrl(:persona)
+              attribute :templated, TrueClass
+            end
+            attribute :customer_config, optional: true
```

Comment:
Why are we making the `customer_config` attribute optional? Shouldn't it be an empty object by default?

Reply:
True